src/api/SFA.DAS.Provider.Events.Infrastructure/Data/DcfsSubmissionEventsRepository.cs

```diff
@@ -79,7 +79,7 @@ public DcfsSubmissionEventsRepository()
         {
             var eventIdFilterClause = eventId > 0 ? $"se.Id > {eventId} AND " : "";
 
-            var command = $@"SELECT* FROM(
+            var command = $@"SELECT *, CommitmentId AS ApprenticeshipId FROM(
                 SELECT ROW_NUMBER() OVER (PARTITION BY se.StandardCode ORDER BY se.Id DESC) rownumber, se.*
                 FROM Submissions.SubmissionEvents se
             WHERE {eventIdFilterClause} se.ULN = {uln}) subEvents  

```

Comment:
Could you remove the string interpolation and instead use a sql parameter. Dapper supports it and it's generally used in the rest of the codebase. It's importand here as this is a user provided field that we're using to access the database

Reply:
"used in the rest of the codebase"

![image](https://user-images.githubusercontent.com/1398567/43399008-a5f2da94-9401-11e8-985c-c6575f5d5c56.png)

Reply:
Done.